src/main/java/uk/gov/hmcts/darts/dailylist/service/impl/DailyListUpdater.java

```diff
@@ -235,7 +247,38 @@ private void addJudges(Sitting sitting, HearingEntity hearing) {
         }
     }
 
-    private String buildFullName(CitizenName citizenName) {
-        return citizenName.getCitizenNameForename() + " " + citizenName.getCitizenNameSurname();
+    private boolean isDefenders(CourtCaseEntity courtCase, PersonalDetails defenders) {
+        boolean existingDefendant = false;
+        for (DefenceEntity defenceEntity : courtCase.getDefenceList()) {
+            if (citizenNameComparator.compare(defenders.getName(), citizenNameMapper.getCitizenName(defenceEntity.getName())) == 0) {
+                existingDefendant = true;
+            }
+        }
+
+        return existingDefendant;
+    }
```

Comment:
Maybe this could be a bit more readable, and the repetition of the `== 0` comparison refactored out? 

I think including "existing" in the method name would help readability of the calling code too: `isExistingDefender()`.

E.g. something like:

```java
    private boolean isExistingDefender(CourtCaseEntity courtCase, PersonalDetails defenders) {
        return courtCase.getDefenceList().stream()
            .anyMatch(defenceEntity -> isEqual(defenders.getName(), citizenNameMapper.getCitizenName(defenceEntity.getName())));
    }

    private boolean isEqual(CitizenName name1, CitizenName name2) {
        return citizenNameComparator.compare(name1, name2) == 0;
    }
```

And the `isEqual()` method could be used in the other similar methods. Just a thought.